doc/z-chap5.xml

```diff
@@ -33,7 +33,13 @@
     <#Include Label="IsDirectedTree">
     <#Include Label="IsUndirectedTree">
     <#Include Label="IsEulerianDigraph">
+    <#Include Label="IsHamiltonianDigraph">
+    <#Include Label="HamiltonianPath">
```

Comment:
This is in the wrong part of the manual, it should be where the other attributes are, not in the section about properties.